ansible/configs/sandbox/default_vars_ec2.yml

```diff
@@ -11,3 +11,5 @@ zone_internal_dns: "{{ guid }}.internal."
 chomped_zone_internal_dns: "{{ guid }}.internal"
 
 infra_ec2_template_generate_auto_select_availability_zone: false
+
+sandbox_enable_console_access: true
```

Comment:
Can we change this to `sandbox_enable_ui`? We already use this to enable console access to IBM Cloud and that is live. If you don't want to change it here, we should change the var we use for IBM Cloud to be the same for consistency.